src/app/tests/suites/commands/system/SystemCommands.cpp

```diff
@@ -77,11 +81,16 @@ CHIP_ERROR SystemCommands::RunInternal(const char * command)
 }
 
 CHIP_ERROR SystemCommands::CreateCommonCommandArgs(char * commandBuffer, size_t commandBufferSize, const char * scriptDir,
```

Comment:
I have future uses for this- the OTA tests that I keep on not being able to get to. Because of https://github.com/project-chip/connectedhomeip/issues/17585, I'll need to create a `SystemCommands::StartOta` or something to that effect which takes pretty much the same arguments as `SystemCommands::Start` but in addition, OTA-specific params. The planned usage is for that new function is to call `CreateCommandArgs` and then append whatever else OTA-specific that gets passed in after.

Please also note I have a fix pending which will use a `StringBuilderBase` rather than `StringBuilder` which could make the appending of the OTA-specific params done easily.